website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -141,6 +158,10 @@ Some example client tokens are:
     ```
     *:development.be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178
     ```
+-   A personal access token with access to everything the user that created them has:
```

Comment:
I don't think we need to repeat how PATs work here. It's mentioned for the other tokens because their form change depending on the details, but if I understand correctly, PATs are always the same: `user:<hash>`, yeah?

```suggestion
-   A personal access token:
```

Reply:
Yeah, they're always the same thing, sounds good, committed!